### ISSUE TITLE
use default github branch instead of master

### ISSUE THREAD

**Issue by SindreSvendby** [codeship/shipscope#13]
shipcode is now in the main window dispalying the status for the master branch. in some of our github repos we do not have master. and have changed the default github branch to be develop. it would be nice to have the ability to either track the defualt github branch or be able to choose which branch is the importent one. for us master most of the time also is prod. and it is the develop branch that is most valuable to se

**Comment by mcgaffin** [comment]
Oh, I see. That's a good suggestion. I'll queue that up!

**Comment by defionscode** [comment]
Yep. Deffo need this as well.

**Comment by VigdisKildedalen** [comment]
That's a good point, Sindre.  This issue is important

**Comment by risto1913** [comment]
Yes I agree. I also work the same way. Hope to see this implemented.

**Comment by mcgaffin** [comment]
Hi,

I will definitely implement this feature. I have a couple other high priority items, but expect to see this in the next 30 days. Thanks for your comments and suggestions.
